Serverless/Azure Functions/About Azure Functions.md

```diff
@@ -1,36 +1,57 @@
+* Newly built functions should be **out-of-process** (you can read more about this in our [In-process and isolated Functions chapter](./In-process-and-isolated-functions)).
 
- A single-function app in the Consumption plan scales out to a maximum of 200 instances, while a single instance can process more than one message/request at a time.
+### Project creation
 
-Timer trigger functions are singleton (trigger only one instance of function).
+Add a new project through the VS Function App template. When choosing the Functions worker, be sure to pick an Isolated one, other workers were applicable for previous versions. A Function project can contain multiple functions that share a common configuration such as environment variables, app settings, and host.
+All functions will be deployed together under the same function-app umbrella and will be scaled together.
 
-In real-world scenarios, we need to control the scaling of instances. Sometimes Azure functions deal with databases or other services which may have scaling limits, so over-scaling Azure functions could put other resources under pressure. Scaling rules can be configured for a function depending on the trigger type.
+### Function app files
 
-Example: If for some reason, we needed queue messages to be processed by a function one at a time then we need to configure the function app to scale out to a single instance. Then the function concurrent processing should be limited too, and that is done by setting `batchSize` to 1 in the `host.json`. Learn more [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-queue-trigger?tabs=csharp#concurrency) and [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-concurrency).
+The default function app consists of:
 
+* `Function.cs` class - Contains a function method with defined triggers and input/output bindings. The `[FunctionName]` attribute marks the method as a function entry point. The name must be unique within a project.
+* `local.settings.json` - Contains all app settings, connection strings, and configurations. Settings in the `local.settings.json` file are used only when you're running the project locally so it is not deployed. Note that when you create a new Function project, this file is added to the `.gitignore` file, meaning that it will not end up in the repository if we don't set that up ourselves. By default, it contains the ``AzureWebJobsStorage`` key with a value set to ``UseDevelopmentStorage=true``, which is the connection string for the local Azure Storage Emulator.
+* `host.json` - Contains the global config options for all functions within a Function app. Here you can configure logging and retry logic. See options in detail [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json#sample-hostjson-file).
+* `Program.cs`  (applicable for out-of-process) - Contains host builder logic, DI configuration and middleware configuration.
```

Comment:
We have multiple spaces after the `Program.cs` and before the `(applicable for out-of-process)`, can we fix that?